Basic-Video-Call/js/app.js

```diff
@@ -0,0 +1,138 @@
+/* global OT API_KEY TOKEN SESSION_ID SAMPLE_SERVER_BASE_URL */
+
+var apiKey;
+var sessionId;
+var token;
+
+// Handling all of our errors here by alerting them
+function handleError(error) {
+  if (error) {
+    alert(error.message);
+  }
+}
+
+
+function toggleStyle(button) { // toggles button css class
+  button.classList.toggle('toggle-button-on');
+  button.classList.toggle('toggle-button-off');
+}
+
+
+function initializeSession() {
+  var session = OT.initSession(apiKey, sessionId);
```

Comment:
As best practice this should be `const` instead of `var`. Even when eslint is es5 configured in this repo, we should follow es6, and eventually update the whole repo. We have it in our backlog, but feel free to update it by your own :)

Reply:
done

Reply:
reverted change - IE does not support neither `const` nor `let`